source/scss/_pf-component-view-selector.scss

```diff
@@ -0,0 +1,17 @@
+//
+// View Selector component
+//
+
+.pf-view-selector__btn-link {
```

Comment:
Related to this, can you review my comment here: https://github.com/patternfly/patternfly-atomic/pull/26/files#r86340374